x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/documents.tsx

```diff
@@ -100,22 +88,51 @@ export const SearchIndexDocuments: React.FC = () => {
     setPagination(DEFAULT_PAGINATION);
     getMappings({ indexName: indexToShow });
   }, [indexToShow]);
-
   return (
-    <EuiPanel hasBorder={false} hasShadow={false} paddingSize="none">
-      <EuiSpacer />
-      <EuiFlexGroup direction="column">
-        <EuiFlexItem>
-          <EuiFlexGroup direction="row" alignItems="center">
-            <EuiFlexItem className="enterpriseSearchDocumentsHeader" grow={false}>
-              <EuiTitle>
-                <h2>
-                  {i18n.translate('xpack.enterpriseSearch.content.searchIndex.documents.title', {
-                    defaultMessage: 'Browse documents',
+    <>
```

Comment:
I don't think we need this `<>` wrapper if it only has the since `< DocumentsOverview />` as a child

Reply:
Yeah, good catch. Will update